src/lib.rs

```diff
@@ -1,6 +1,9 @@
 #[macro_use]
 extern crate redis_module;
 
+#[macro_use]
+extern crate log;
```

Comment:
This syntax is deprecated. Instead of it, just `use log::debug;` in the relevant file(s).

And while you're at it, we can remove the above `#[macro_use] extern crate redis_module;` and replace it with `use redis_module::{redis_command, redis_module};`.

Reply:
Removed